spec/requests/edit_collection_spec.rb

```diff
@@ -47,14 +45,8 @@
               "9995" => {"sunetid" => "giancarlo", "_destroy" => "false"},
               "9994" => {"sunetid" => "zhengyi", "_destroy" => "false"}
             },
-            email_depositors_status_changed: true,
-            contact_emails_attributes: {}
-          }.tap do |param|
-            collection_version.contact_emails.each_with_object(param[:contact_emails_attributes])
-              .with_index do |(author, attrs), index|
-              attrs[index.to_s] = {"_destroy" => "false", "id" => author.id, "email" => "bob@foo.io"}
-            end
```

Comment:
Also for collection versions rather than collections. This code likely pre-dates the existence of the distinction between colls and collvers.